src/components/data-processing/CalculationConfigContainer.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Space,
+  Form,
+  Button,
+  Alert,
+  Radio,
+} from 'antd';
+
+import _ from 'lodash';
+
+import { updateProcessingSettings } from '../../redux/actions/experimentSettings';
+
+const CalculationConfigContainer = (props) => {
+  const {
+    filterUuid, experimentId, sampleId, plotType, sampleIds, onConfigChange, children,
```

Comment:
what is the difference between`sampleId` and `sampleIds`? Isn't `sampleId` included in `sampleIds`? Why do we need to have them both?

Reply:
The `Copy to all samples` button needs to be able to update the rest of the samples
![image](https://user-images.githubusercontent.com/460418/112980479-3b207880-915a-11eb-8b05-0a8d5dc91572.png)

Reply:
I see, so when the user clickes `copy to all samples`, then we use `sampleIds`, otherwise we use `sampleId`?

Reply:
Right